dev/tasks/python-wheels/osx-build.sh

```diff
@@ -183,14 +183,19 @@ function install_wheel {
     pip install $(pip_opts) \
         $(python $multibuild_dir/supported_wheels.py $wheelhouse/*.whl)
 
-    # Install test dependencies
-    pip install $(pip_opts) -r python/requirements-test.txt
     popd
 }
 
 function run_unit_tests {
+    pushd $1
+
+    # Install test dependencies
+    pip install $(pip_opts) -r python/requirements-test.txt
+
     # Run pyarrow tests
-    py.test --pyargs pyarrow
+    pytest -rs --pyargs pyarrow
```

Comment:
Why is this without a `.` in the name?

Reply:
One is just an alias for the other.

Reply:
I think we generally use the "pytest" spelling. At least that's the one I use locally (it's also consistent with "python -m pytest").

Reply:
`pytest` is the preferred command since a couple of releases now, the documentation also refers that https://docs.pytest.org/en/latest/usage.html

Reply:
`py.test` was once the preferred way but browsing the documentation, it seems this is no longer the case.